### ISSUE TITLE
Caddyfile support backtick strings

### ISSUE THREAD

**Issue by caibirdme** [caddyserver/caddy#2591]
## 1. What would you like to have changed?

Caddyfile support multi line string

## 2. Why is this feature a useful, necessary, and/or important addition to this project?

without supporting of multi line string, it's difficult to develop some new plugins which need user to configure some complex string such as json.

## 3. What alternatives are there, or what are you doing in the meantime to work around the lack of this feature?

```
directive here_might_be_a_json

# now
directive here_json_file_path
```

**Comment by caibirdme** [comment]
If you approve this feature, I am willing to contribute

**Comment by mholt** [comment]
I believe the Caddyfile already supports multiline strings when enclosed in quotes.

**Comment by caibirdme** [comment]
@mholt I mean somthing like this:
```
`{
"name": "foo",
 "age": "bar",
}`
```
But now, we must do it like:
```
"
{\"name\":\"foo\",
\"age\":\"bar\"
}
"
```

**Comment by linquize** [comment]
Don't want to escape string

**Comment by mholt** [comment]
Ah, I see. So you want an alternate strings delimiter. Definitely possible.

**Comment by caibirdme** [comment]
Yes, it would be perfect if Caddyfile supports this

**Comment by mholt** [comment]
Feel free to submit a PR, I hope the implementation will not be too tricky!

Also consider if this could break anyone (I doubt it, but, worth considering.)

**Comment by caibirdme** [comment]
I'll try my best

**Comment by vaibhav2ghadge** [comment]
@mholt what happen about this feature..? if not implemented yet can you explain me what we wan to do.

**Comment by mholt** [comment]
I dunno, I never got a PR from @caibirdme -- I don't have the time to do it right now but you're welcome to submit a PR for review.

**Comment by vaibhav2ghadge** [comment]
@mholt can you explain what have to do. i mean which feature your expecting

**Comment by mholt** [comment]
As described above, I think multi-line strings delimited by ` is a good idea:

```
`{
"name": "foo",
 "age": "bar",
}`
```

**Comment by vaibhav2ghadge** [comment]
@mholt im not able to understand for which scenario this will be usefull i mean when this type of object will come from whom

**Comment by mholt** [comment]
Implemented in #3242, will go out by Caddy 2.1.
